packages/main/src/RPA/Smartsheet.py

```diff
@@ -0,0 +1,1415 @@
+            sheet_id,
+            include=includes,
+            row_ids=self._parse_comma_list_type(row_ids),
+            row_numbers=self._parse_comma_list_type(row_numbers),
+            exclude="filteredOutRows",
+            column_ids=self._parse_comma_list_type(column_ids),
+            filter_id=filter_id,
+        )
+        setattr(sheet, "selected_includes", includes)
+        if download_path is not None:
+            setattr(sheet, "download_path", download_path)
+        self.current_sheet = sheet
+        if native:
+            return sheet
+        else:
+            return self.convert_sheet_to_table(sheet)
+
+    @keyword
+    def convert_sheet_to_table(self, sheet: Sheet = None) -> Table:
+        """Converts the current sheet to table. You can provide a differnt
+        native sheet object to be converted via the ``sheet`` parameter.
+
+        This keyword attempts to return the sheet as a table via
+        ``RPA.Tables``, but if that library is not available in this
+        context, the sheet is returned as it's native data model (e.g.,
+        no operation is performed).
+
+        If the sheet contains additional data from the ``include``
+        argument, they will be appended to the end of the table as
+        additional columns in the data model. These additional objects
+        will be attached as a list of objects depending on the items
+        requested.
+
+        .. warn:
+
+            Only the following supported additional data requests
+            are supported when unpacking a Sheet object.
+
+            * ``attachments``: The value for the cell will be a list of
+            native Attachment object.
+            * ``attachmentFiles``: The value for the cell will be a list of
+            file paths to the local copies of each attachment. This
+            operation may be resource intensive for many attachments.
+            * ``discussions``: The value for the cell will be a list of
+            native Discussion objects.
+            * ``rowPermalink``: The value for the cell will be a single
+            string representing the URL to the permalink.
+        """
+        if sheet is None:
+            self._require_current_sheet()
+            sheet = self.current_sheet
+        includes = getattr(sheet, "selected_includes", []) or []
+        table_data = []
+        headers = []
+        for column in sheet.columns:
+            headers.append(column.title)
+        if len(includes) > 0:
+            headers.extend(includes)
+
+        for row in sheet.rows:
+            row_data = []
+            for cell in row.cells:
+                row_data.append(cell.value)
+            for key in includes:
+                key_or_func = self.SHEET_INCLUDES[key]
+                if callable(key_or_func):
+                    row_data.append(key_or_func(row))
+                elif getattr(row, key_or_func, False):
+                    row_data.append(getattr(row, key_or_func, None))
```

Comment:
```suggestion
                    row_data.append(getattr(row, key_or_func))
```
if we get here then we're dead sure that this found the attribute, right?

Reply:
Yes, the underlying native object should have that key as an attribute, if it doesn't, then it's malformed.